pkg/model/api/v1/variable/list.go

```diff
@@ -67,23 +67,23 @@ func (v *DefaultValue) MarshalYAML() (interface{}, error) {
 
 type ListSpec struct {
 	Display       *Display      `json:"display,omitempty" yaml:"display,omitempty"`
-	DefaultValue  *DefaultValue `json:"default_value,omitempty" yaml:"default_value,omitempty"`
-	AllowAllValue bool          `json:"allow_all_value" yaml:"allow_all_value"`
```

Comment:
@AntoineThebaud If we do move forward with this proposal, would `json` and `yaml` still be the same casing? I did a find & replace but want to confirm this diff is correct?

Reply:
Yes they would & should remain identical.